### DISCUSSION TITLE
Knex Sql Error

### DISCUSSION THREAD

**Discussion by uyanikislam** [directus/directus#12987] [Archive]
Hello,
When I load directus and after write to 'npx directus start' (windows, nodejs), ⬇️ thats happening. What was my fault? Please help me. Thank you.

TypeError: String expected
    at C:\Users\sarga\my-projecto\node_modules\knex\lib\dialects\sqlite3\index.js:85:18
    at new Promise (<anonymous>)
    at Client_SQLite3.acquireRawConnection (C:\Users\sarga\my-projecto\node_modules\knex\lib\dialects\sqlite3\index.js:69:12)
    at create (C:\Users\sarga\my-projecto\node_modules\knex\lib\client.js:247:39)

**Comment by uyanikislam** [comment]


**Reply by rijkvanzanten** [reply]
What was the problem, and what was the fix?

**Reply by uyanikislam** [reply]
> What was the problem, and what was the fix?

I dont know problem but it doesnt fixed :( i choosed sql database and this happened.

**Reply by uyanikislam** [reply]
> What was the problem, and what was the fix?

https://github.com/directus/directus/issues/10022

Same problem is here from another person sir.
